server/recruitment.py

```diff
@@ -35,6 +35,7 @@ def recruiter_claim_applicant(recruiter_user_id, applicant_user_id):
         applicant_name = Character.get(applicant_user_id).name
         return {'error': 'User {} is not an applicant'.format(applicant_name)}
     applicant.recruiter_id = recruiter_user_id
+    applicant.status_level = 1
```

Comment:
Use applicant.status = 'escalated' for things like this, I made a nice setter for it that would set status_level if you do that. But again, let's not duplicate information with a status level of `claimed`.

Reply:
see reason for claimed above
I'd like to set a status something like `status_level = statuses.escalated` rather than a number (subject to change and then error) or string (subject to typo)

Reply:
You can remove the applicant.status_level line here with the new changes to the User model

Reply:
done